src/test/java/jewellery/inventory/integration/UserCrudIntegrationTest.java

```diff
@@ -35,7 +33,7 @@ private String getBaseUserUrl() {
   @Test
   void createUserSuccessfully() throws Exception {
 
-    UserRequestDto userRequest = createTestUserRequest();
+    UserRequestDto userRequest = createDifferentTestUserRequest();
```

Comment:
I think all these tests shouldn't be changed if we put a different admin user as a currently authenticated user